apps/fhir/bluebutton/tests/test_read.py

```diff
@@ -601,3 +604,67 @@ def catchall(url, req):
                 Authorization="Bearer %s" % (first_access_token))
 
             self.assertEqual(response.status_code, 200)
+
+    def test_application_first_last_active(self):
+        # create the user
+        first_access_token = self.create_token('John', 'Smith')
+
+        application = Application.objects.get(pk=1)
```

Comment:
this makes this test a bit brittle, if you want the one associated with `first_access_token` see if you can do that explicitly. If you're looking for the first application see if you can do that more explicitly.

Reply:
I see. Will change to grab it by the application name instead of pk=1. That create_token() only returns back the token's string and not the object.

Reply:
Updated to get the application object via the related access token object.